nexus/tests/integration_tests/endpoints.rs

```diff
@@ -1246,7 +1249,7 @@ pub static DEMO_WEBHOOK_SECRET_CREATE: LazyLock<params::WebhookSecretCreate> =
         secret: "TRUSTNO1".to_string(),
     });
 
-// pub static DEMO_WEBHOOK_SUBSCRIPTION: LazyLock<shared::WebhookSubscription> =
+// pub static DEMO_ALERT_SUBSCRIPTION: LazyLock<shared::WebhookSubscription> =
```

Comment:
Nit: I know this is commented out code (why is this here?) but should this be a `shared::AlertScubscription` now?